tests/components/hue/test_services.py

```diff
@@ -49,11 +50,8 @@
 
 async def test_hue_activate_scene(hass: HomeAssistant, mock_api_v1) -> None:
     """Test successful hue_activate_scene."""
-    config_entry = config_entries.ConfigEntry(
-        version=1,
-        minor_version=1,
```

Comment:
You're losing out on `minor_version=1`, does that matter ?

Reply:
It's the default, it was only added because `ConfigEntry` requires passing the minor version, the point of this PR is to not have to update the `hue` tests when we make changes to config entries.